packages/rich-text/src/get-parent-line-index.js

```diff
@@ -14,8 +14,8 @@ import { LINE_SEPARATOR } from './special-characters';
  *
  * @return {Array} The parent list line index.
  */
-export function getParentLineIndex( { text, formats }, lineIndex ) {
-	const startFormats = formats[ lineIndex ] || [];
+export function getParentLineIndex( { text, replacements }, lineIndex ) {
+	const startFormats = replacements[ lineIndex ] || [];
```

Comment:
Variable should be renamed to reflect the fact it doesn't consume from `formats` ?

(Same note below for `formatsAtIndex`)

(There are a couple other occurrences of this throughout the pull request)